__tests__/comment.test.js

```diff
@@ -99,3 +99,102 @@ describe("GET /api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  const content = {
```

Comment:
It's best to include this content afresh in each test - for readability and to prevent errors caused by accidental mutation in our tests